### DISCUSSION TITLE
Importing Layerzero Smart contracts

### DISCUSSION THREAD

**Discussion by 0xtomoon** [foundry-rs/foundry#9279] [Q&A]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0

### What command(s) is the bug in?

forge test -vvv

### Operating System

macOS (Intel)

### Describe the bug

I use openzeppelin library with version 5.0.2, but when I import [@layerzerolabs/lz-evm-messagelib-v2](https://github.com/LayerZero-Labs/LayerZero-v2/blob/main/packages/layerzero-v2/evm/messagelib/package.json#L13) it uses version 4.8.1 inside

How can I handle remapping in this case?

**Comment by mattsse** [comment]
can you take a look at your remappings `forge re`

this is likely a problem with autodetection, you can disable that with auto_detect_remappings = false

**Comment by grandizzy** [comment]
@0xtomoon can you provide input to comment above? thank you!
